### PR TITLE
Add endpoint

### PR DESCRIPTION
**Pull request by h8570rg**
## Summary

before
localhost_8000_ja_dashboard_endpoints_ (1)

after

localhost_8000_ja_dashboard_endpoints_

### CODE REVIEW

File: packages/app/src/components/button/filled/index.tsx [hunk 1]
```diff
@@ -114,15 +114,10 @@ const Renewal = function <T = null>({
...
+        `flex items-center gap-1 px-3.5 bg-thm-${cs} hover:opacity-75 active:opacity-50 focus:outline outline-thm-outline text-thm-on-${cs}`,
         {
           'rounded-full': rounded,
-          'text-xxs': size === SIZE.XXS,
-          'text-xs': size === SIZE.XS,
-          'text-sm': size === SIZE.SM,
-          'text-base': size === SIZE.BASE,
-          'text-xl': size === SIZE.XL,
-          'text-2xl': size === SIZE['2XL'],
+          'text-sm h-8': size === SIZE.SM,
```
Comment (h8570rg) [inline packages/app/src/components/button/filled/index.tsx:120]: Other sizes are not determined. I will fill it in when design created.

File: packages/app/src/components/button/icon/on/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import classnames from 'classnames';
+import React, { useCallback } from 'react';
+import { Props as BaseProps } from '~/components';
+import { Props as BaseButtonProps, SIZE } from '~/components/button';
+
+export type Props<T = null> = BaseProps & BaseButtonProps<T>;
+const IconButton = function <T = null>({
```
Comment (h8570rg) [inline packages/app/src/components/button/icon/on/index.tsx:7]: New component for icon button like modal close button.

File: packages/app/src/components/textinput/index.tsx [hunk 3]
```diff
@@ -53,4 +54,47 @@ const Textinput: React.FC<Props> = ({
...
+            <label htmlFor={id} className="text-sm font-bold block">
+              {label}
+            </label>
+          )}
+          {!!description && <p>{description}</p>}
+        </div>
+      )}
+      {render(bind)}
+      {!!error && (
+        <p className="text-xxs">{t(`validation.${error.message}`)}</p>
```
Comment (h8570rg) [inline packages/app/src/components/textinput/index.tsx:92]: There are no I18n values for validation for now. I will make it later.

File: packages/app/src/components/button/icon/on/index.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    onClick(data as T);
+  }, [data, onClick]);
+
+  return (
+    <button
+      className={classnames(
+        `flex items-center justify-center disabled:pointer-events-none disabled:opacity-60 transition-color hover:bg-thm-on-${on}-slight active:bg-thm-on-${on}-low outline-thm-outline`,
+        className,
+        {
+          'h-11 w-11 text-xl rounded': size === SIZE.BASE,
```
Comment (nonoakij) [inline packages/app/src/components/button/icon/on/index.tsx:27]: I think whether 'rounded' is added or not should be determined by the rounded props, not the size props.
```suggestion
          'rounded-full': rounded,
          'h-11 w-11 text-xl': size === SIZE.BASE,
```

File: packages/app/src/components/select/index.tsx [hunk 5]
```diff
@@ -48,4 +49,47 @@ const _Select = function <T = unknown>({
...
+      )}
+      <Select
+        id={id}
+        className={`block w-full h-10 px-3 border rounded-lg border-thm-on-${on}-low bg-thm-${on} text-thm-on-${on} outline-thm-outline`}
+      >
+        {OptionBlank && <OptionBlank className="p-1" />}
+        {list.map((item, idx) => (
+          <React.Fragment key={idx}>
+            <Option className="p-1" data={item} />
+          </React.Fragment>
```
Comment (nonoakij) [inline packages/app/src/components/select/index.tsx:86]:
```suggestion
          <Option key={idx} className="p-1" data={item} />

```

File: packages/app/src/components/textinput/index.tsx [hunk 6]
```diff
@@ -53,4 +54,47 @@ const Textinput: React.FC<Props> = ({
...
+    bind.list = autocompleteId;
+  }
+  return (
+    <div className={classnames(`text-thm-on-${on}`, className)}>
+      {(!!label || !!description) && (
+        <div className="space-y-1 mb-3">
+          {label && (
+            <label htmlFor={id} className="text-sm font-bold block">
+              {label}
+            </label>
```
[the remark this answers is not in the document]
Comment (h8570rg) [reply packages/app/src/components/textinput/index.tsx:85]: Then what is the content of label tag?
aria-label or label has to be passed from parent, isn't it? 
Passing aria-label or label will  take a high cost now.

File: packages/app/src/components/button/icon/on/index.tsx (hunk 4 above)
Comment (h8570rg) [reply packages/app/src/components/button/icon/on/index.tsx]: fb44a2285402fafb47ec6040c762ef1768ab481d
Fixed!

File: packages/app/src/components/textinput/index.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (nonoakij) [reply packages/app/src/components/textinput/index.tsx:85]: > Then what is the content of label tag?

Take the string that describes `<Select>` as `<label>` and associate `<label>` with `<Select>` using htmlFor props.
"sr-only" can be used to convey a message only to screen readers without displaying it on the screen.

> Passing aria-label or label will take a high cost now.
Why do you think it high costs?

[the remark this answers is not in the document]
Comment (nonoakij) [reply packages/app/src/components/textinput/index.tsx:85]: I expect the implementation to be as follows
``` tsx
      <Select
        id={id}
        className={`block w-full h-10 px-3 border rounded-lg border-thm-on-${on}-low bg-thm-${on} text-thm-on-${on} outline-thm-outline`}
        labelSrOnly
        label="Please select the group to include this endpoint"
      >
        {OptionBlank && <OptionBlank className="p-1" />}
        {list.map((item, idx) => (
          <Option key={idx} className="p-1" data={item} />

        ))}
      </Select>
```
``` tsx
    <div className={classnames(`text-thm-on-${on}`, className)}>
      {(!!label || !!description) && (
        <div className="space-y-1 mb-3">
            <label htmlFor={id} className={classNames("text-sm font-bold block", {'sr-only': labelSrOnly })} >
              {label}
            </label>
```

[the remark this answers is not in the document]
Comment (h8570rg) [reply packages/app/src/components/textinput/index.tsx]: ff7920d481e0a80835d57ac9231461d861b16fc0
I made label props required only in renewaled component!

**Review by nonoakij — APPROVED** [review]

**Pull request merged**